public/js/googleMaps.js

```diff
@@ -0,0 +1,172 @@
+/******/ })
+/************************************************************************/
+/******/ ({
+
+/***/ "./resources/js/googleMaps.js":
+/*!************************************!*\
+  !*** ./resources/js/googleMaps.js ***!
+  \************************************/
+/*! no static exports found */
+/***/ (function(module, exports) {
+
+function initAutocomplete() {
+  new google.maps.places.Autocomplete(document.getElementById('locationName'));
+}
+
+function initMap() {
+  var map = new google.maps.Map(document.getElementById("map"), {
+    zoom: 8,
+    center: {
+      lat: 44.4267674,
+      lng: 26.1025384
+    }
+  });
+  var locationAddress = document.getElementById('locationForm');
+  locationAddress.addEventListener('submit', geocode);
+
+  function geocode(e) {
+    //Previne trimiterea form-ului
+    e.preventDefault();
+    var location = document.getElementById('locationName').value;
+    axios.get("https://maps.googleapis.com/maps/api/geocode/json", {
+      params: {
+        address: location,
+        key: 'AIzaSyBB0Tj2YoWLm7ozC9I9P_zkESQeRmCKDXw'
+      }
+    }).then(function (response) {
+      //Afisam tot raspunsul primit in consola
+      console.log(response); //Locatia
+
+      var lat = response.data.results[0].geometry.location.lat;
+      var lng = response.data.results[0].geometry.location.lng;
+      var afisareaLatLng = "\n            <ul>\n                <li>".concat(lat, "</li>\n                <li>").concat(lng, "</li>\n            </ul>\n            "); //Adresa formatata
+
+      var adresaFormatata = response.data.results[0].formatted_address;
+      var afisareadresaFormatata = "\n            <ul>\n                <li> ".concat(adresaFormatata, "</li>\n            </ul>\n            "); //Afisam pe ecran
+
+      document.getElementById('adresaFormatata').innerHTML = afisareadresaFormatata;
+      document.getElementById('position').innerHTML = afisareaLatLng; //Centram harta acolo unde a bagat utilizatorul
+
+      map.setCenter({
+        lat: response.data.results[0].geometry.location.lat,
+        lng: response.data.results[0].geometry.location.lng
+      }); // The marker
+
+      var marker = new google.maps.Marker({
+        position: {
+          lat: response.data.results[0].geometry.location.lat,
+          lng: response.data.results[0].geometry.location.lng
+        },
+        map: map
+      });
+    })["catch"](function (error) {
+      console.log(error);
+    });
+  }
+
+  initAutocomplete();
+}
+
+window.onload = function () {
+  initMap();
+};
+
+/***/ }),
+
+/***/ 1:
+/*!***************************************!*\
+  !*** multi ./resources/js/googleMaps ***!
+  \***************************************/
+/*! no static exports found */
+/***/ (function(module, exports, __webpack_require__) {
+
+module.exports = __webpack_require__(/*! C:\wamp\proiecte\cartelProject\resources\js\googleMaps */"./resources/js/googleMaps.js");
+
+
+/***/ })
+
+/******/ });
```

Comment:
please fix this code in this file, and by fixing I mead: remove anything that is in romanian, and be careful about indentation, also the webpack part shouldn't be in this file

Reply:
These are automatically generated by the webpack when i run npm run dev.Why this should not be here?